### ISSUE TITLE
Proposal: remove background-color from .mat-drawer-container

### ISSUE THREAD

**Issue by adamdport** [angular/material2#7536]
#### Bug, feature request, or proposal:
Proposal: remove `background-color` from `.mat-drawer-container`

#### What is the expected behavior?
The common background that I've set in my app appears on a page with a drawer

#### What is the current behavior?
Putting a drawer on a page requires putting the page contents inside `md-drawer-container` which comes with `background-color` set that overrides any app background that's been set.

#### What are the steps to reproduce?
Add a drawer to a page with a non-standard background

#### What is the use-case or motivation for changing an existing behavior?
Overlays for drawers and sliders are typically supposed to block the majority of the UI. IMHO material shouldn't be providing a background-color for this majority.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
2.0.0-beta.10

**Comment by ja5onkendall** [comment]
I agree with @adamdport I have to override this background color every time.

**Comment by rtaft** [comment]
You might as well add .color to that as well as it overrides both

**Comment by DjStorm** [comment]
PLease do it.

**Comment by wintersommer** [comment]
thats what is there currently , why ?

    background-color: #fafafa;
    color: rgba(0,0,0,.87);

**Comment by hcurnor** [comment]
```
.mat-drawer-container{
background-color: #fafafa;
color: rgba(0,0,0,.87);
}
```
Its reflecting on my svgs and other elements too (in safari).
It should be removed

**Comment by upondeep** [comment]
This mat-drawer css is broken. It even overwrites my "routerLinkActive" color.

**Comment by HuiiBuh** [comment]
A fix would be really appreciated. You cant offer theming and say that the components support it and then you have to set the background color of some components manually

**Comment by paco76** [comment]
I asked this question on stackoverflow, maybe it can help..

```
<mat-drawer-container class="red-container">
...
</mat-drawer-container>

.red-container {
  background-color: red;
}

**Comment by rickdgray** [comment]
These seems like a trivial fix. Has there been any movement on this?

**Comment by michahell** [comment]
This can cause very slight visual regressions in existing projects where background colors are just #FFF white, and makes it so that a first experience of implementing an Angular Material component requires a style FIX, as opposed to a style customization. Which in turn does not provide a good first experience, which is a bummer. Why is it there?

**Comment by RamZal365** [comment]
This is still open right? Not the first time I end up in this issue. I would like to also suggest to add the possibility of setting the color of the drawer using the app theme e.j. (color="primary")

**Comment by SubJunk** [comment]
The hardcoded background color was removed in v17 so I think this issue can be closed. You can see it is gone by using the StackBlitz examples in their v17 docs, like this one https://stackblitz.com/run?file=src%2Fexample%2Fsidenav-drawer-overview-example.ts
Note that in that example, the background color is being customized and coming from the file `sidenav-drawer-overview-example.css`, not from AM itself.
